.changeset/afraid-guests-relax.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+Converted `SuperSelect` components to TypeScript
```

Comment:
Nit: most of the other ts conversion change sets have been like:
```md
`Table` - Converted component and sub-components to TypeScript
```

Reply:
Updated in https://github.com/hashicorp/design-system/pull/2356/commits/217c865c29b101cebdb60b7f19fd91be31f57dac